github_actions/lib/dependabot/github_actions/package_manager.rb

```diff
@@ -0,0 +1,95 @@
+# typed: strong
+# frozen_string_literal: true
+
+require "sorbet-runtime"
+require "dependabot/github_actions/version"
+require "dependabot/ecosystem"
+require "dependabot/github_actions/requirement"
+
+module Dependabot
+  module GithubActions
+    DOTCOM = "github.com"
```

Comment:
Do you expect this URL to change sometime? If so, the naming shouldn't be `DOTCOM` because the domain may _not_ use the `.com` TLD... perhaps something more like `DOMAIN_ROOT` or `ACTIONS_HOST`? 🤷‍♂️ 

But if you don't expect it to change, then I don't see the value in this abstraction? `DOTCOM` is a pretty generic name and the code would be more readable with `github.com` inlined... or if there's really a need for a constant but don't expect it to change, then what about using `GITHUB_COM`?